cli/tsup.config.ts

```diff
@@ -4,12 +4,9 @@ const isDev = process.env.npm_lifecycle_event === "dev";
 
 export default defineConfig({
   clean: true,
-  dts: true,
```

Comment:
also no use to ship type declarations with a CLI, just extra space the user has to download...